libkbfs/identify_util.go

```diff
@@ -5,68 +5,139 @@
 package libkbfs
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/protocol/keybase1"
 	"golang.org/x/net/context"
 )
 
-func identifyUID(ctx context.Context, nug normalizedUsernameGetter, identifier identifier, uid keybase1.UID, isPublic bool) error {
+func identifyUID(ctx context.Context, nug normalizedUsernameGetter,
+	identifier identifier, uid keybase1.UID, isPublic bool) error {
+	_, _, err := identifyUIDWithIdentifyBehavior(ctx,
+		nug, identifier, uid, isPublic, keybase1.TLFIdentifyBehavior_DEFAULT_KBFS)
+	return err
+}
+
+func identifyUIDWithIdentifyBehavior(ctx context.Context,
```

Comment:
Having a double "identify" in the name is confusing.  Maybe just `identifyUIDWithBehavior`?